Vagrantfile

```diff
@@ -113,7 +113,7 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provider "virtualbox" do |vb|
-    vb.memory = "3196"
+    vb.memory = "6144"
```

Comment:
This doubling of memory could make the physical machine run out of memory.

Please add a section to vagrant.md about number of hosts, memory usage so user can configure for the amount of memory he/she has.